src/Context.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Context.h"
+
+#include "licenses/LicenseCategory.h"
+
+Context::Context() {
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelWebView, licenses::LicenseCategory::WebView);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelToolkit, licenses::LicenseCategory::Toolkit);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelPlatform, licenses::LicenseCategory::Platform);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelAll, licenses::LicenseCategory::All);
+}
+
+void Context::connectSearchModel(licenses::LicenseModel * model, licenses::LicenseFilter & search_model, licenses::LicenseCategory category) {
```

Comment:
why is model a pointer instead of reference?

Reply:
No good reason :D a leftover after a few rewrites, fixed now, thank you!